torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -169,7 +170,9 @@ struct DependencyChains : public IterVisitor {
       traverse(_dependency->fusion(), false);
   }
 
-  DependencyChains(std::set<Val*> _dependencies, bool all_chains_ = false)
+  DependencyChains(
+      const std::set<Val*>& _dependencies,
```

Comment:
This will break clang-tidy.
We should revert the changes.

Reply:
Ugh clang-tidy how I hate you.